### PR TITLE
[MS] Added user contact details page with password change

### LINKED ISSUE #5599: Add password change page
And mock.

Add mock-ups.

### LINKED ISSUE #5697: Add current user contact details page
Something like:
image

Linked to #5599

### PR DESCRIPTION
**Pull request by Max-7**
Closes #5697
Closes #5599 

Design is a bit rough, it needs to be cleaned `a bit` before merging.

contact-details

- [X] Keep changes in the pull request as small as possible
- [X] Ensure the commit history is sanitized
- [X] Give a meaningful title to your PR
- [X] Describe your changes
- [X] Link any related issue in the description

### CODE REVIEW

File: client/src/components/core/ms-input/MsPasswordInput.vue
Comment (Max-7) [inline client/src/components/core/ms-input/MsPasswordInput.vue:1]: Updates on this file may seem weird, but it's to fix the two-way data bindings. Basically, when using `v-model` like
```
<ms-password-input
  v-model="password"
>

const password = ref('');
```
`password` would take the right value when text was entered in the input, but when doing `password.value = 'my text'`, the input would not be updated. This should fix that.

File: client/src/locales/fr-FR.json [hunk 1]
```diff
@@ -727,5 +730,16 @@
...
     "Notification": {
         "nextButton": "Okay !"
+    },
+    "ContactDetailsPage": {
+        "avatarTitle": "Avatar",
+        "avatarUnavailable": "Non disponible",
+        "changePasswordButton": "Mettre à jour le mot de passe",
+        "errors": {
+            "failedToRetrieveInformation": "Impossible de récupérer les informations.",
+            "wrongPassword": "Mauvais mot de passe !",
```
Comment (NicoTuxx) [inline client/src/locales/fr-FR.json:740]:
```suggestion
            "wrongPassword": "Mot de passe incorrect !",
```

**Review by NicoTuxx — CHANGES_REQUESTED** [review]

File: client/src/views/header/ProfileHeaderPopover.vue [hunk 2]
```diff
@@ -7,6 +7,18 @@
...
+    <ion-item
+      class="container__item"
+      @click="onOptionClick(ProfilePopoverOption.MyContactDetails)"
+    >
+      <ion-icon
+        :icon="personCircle"
+        slot="start"
+      />
+      <ion-label class="body">
+        {{ 'My contact details' }}
```
[the remark this answers is not in the document]
Comment (fabienSvtr) [reply client/src/views/header/ProfileHeaderPopover.vue:19]: No worries, it's done!

File: client/src/components/users/TagProfile.vue [hunk 3]
```diff
@@ -23,15 +23,15 @@ defineProps<{
   profile: UserProfile
 }>();
 
-function translateProfile(profile: UserProfile): string {
+function translateProfile(profile: UserProfile): { name: string, class: string } {
```
[the remark this answers is not in the document]
Comment (fabienSvtr) [reply client/src/components/users/TagProfile.vue]: Done!

File: client/src/components/users/TagProfile.vue [hunk 4]
```diff
@@ -33,6 +33,17 @@ function translateProfile(profile: UserProfile): string {
...
+
+function getClassForProfile(profile: UserProfile): string {
+  if (profile === UserProfile.Admin) {
+    return 'admin';
+  } else if (profile === UserProfile.Standard) {
+    return 'standard';
+  } else if (profile === UserProfile.Outsider) {
+    return 'outsider';
+  }
+  return '';
```
Comment (NicoTuxx) [inline client/src/components/users/TagProfile.vue:45]: works but this case is clearly a switch case statement, should be replaced by something like:
```
switch (profile) {
  case UserProfile.Admin:
    return 'admin';
  case UserProfile.Standard:
    return 'standard';
  case UserProfile.Outsider:
    return 'outsider';
  default:
    return '';
}
```

File: client/src/locales/fr-FR.json [hunk 5]
```diff
@@ -510,6 +512,8 @@
     },
     "Password": {
         "password": "Mot de passe",
+        "currentPassword": "Mot de passe actuel",
+        "newPassword": "Nouveau mot de passe",
         "confirmPassword": "Confirmer le mot de passe",
         "passwordInfo": "Votre mot de passe doit être sécurisé et vous devez le mémoriser. Nous vous conseillions de créer plusieurs appareils si possible et de partager les espaces de travail.",
```
Comment (NicoTuxx) [inline client/src/locales/fr-FR.json:518]:
```suggestion
        "passwordInfo": "Votre mot de passe doit être sécurisé et vous devez le mémoriser. Nous vous conseillons de créer plusieurs appareils si possible et de partager les espaces de travail.",
```

File: client/src/views/users/MyContactDetailsPage.vue [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+            />
+            <ion-chip class="caption-caption avatar-unavailable">
+              {{ $t('ContactDetailsPage.avatarUnavailable') }}
+            </ion-chip>
+          </div>
+
+          <!-- change password -->
+          <div class="password-change">
+            <h3 class="title-h3 password-change-title">
+              {{ 'Change your password' }}
```
Comment (NicoTuxx) [inline client/src/views/users/MyContactDetailsPage.vue:36]: Should use vue-i18n locale files.

**Review by NicoTuxx — CHANGES_REQUESTED** [review]
2 minor wording problems:
- [x] line 522 of fr-FR.json: `éviter les répétitions` should be replaced by `évitez les répétitions`

File: client/src/locales/fr-FR.json (hunk 5 above)
Comment (fabienSvtr) [reply client/src/locales/fr-FR.json:518]: Done

File: client/src/components/users/TagProfile.vue (hunk 4 above)
Comment (fabienSvtr) [reply client/src/components/users/TagProfile.vue:45]: Done

**Pull request merged**